airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java

```diff
@@ -42,7 +45,7 @@ public SnowflakeInternalStagingSqlOperations(final NamingConventionTransformer n
 
   @Override
   public String getStageName(final String namespace, final String streamName) {
-    return nameTransformer.applyDefaultCase(String.join("_",
+    return nameTransformer.applyDefaultCase(String.join(".",
```

Comment:
@davinchia (and others looking at the PR) - This was the change that I made to address failing tests. The behavior I was seeing was snowflake choosing a "default" schema as this combining by "_" was just a valid stage name not attributed deterministically to a schema.

Reply:
where alternatively the "." join is interpreted as `{schema}.{stage_name}`

Reply:
don't fully understand the implications here. seems fine to me if the purpose is consistency with our test expectations.

users shouldn't look at the staging area often so I'd imagine this is a rather low risk change.